packages/terra-form-select/translations/en-GB.json

```
@@ -27,5 +27,6 @@
   "Terra.form.select.option": "Options",
   "Terra.form.select.optGroup": "Group {text}",
   "Terra.form.select.defaultComboboxDisplay":	"Select or Enter",
-  "Terra.form.select.resultsText": "Results that contain \"{text}\""
+  "Terra.form.select.resultsText": "Results that contain \"{text}\"",
+  "Terra.form.select.deselect": "Deselect"
```

```suggestion
  "Terra.form.select.deselect": "Deselect {text}"
```

Make this change for all translation files after updating the formatMessage